README.md

```diff
@@ -122,37 +126,42 @@ inside.
 
 - Build Settings Externalized
 
-    We iterate all the `BuildConfiguration` blobs, and investigate their
-    `BuildSettings` entry. Empty settings are A-OK. Any found settings are in
-    error.
+We iterate all the `BuildConfiguration` blobs, and investigate their
+`BuildSettings` entry. Empty settings are A-OK. Any found settings are in
+error.
+
+- Dangling Source Files
+
+We iterate all source files appearing in Project Navigator and check all are associated
+with a target. Best used with tests. Any found dangling source file is in error.
```

Comment:
suggestion:

> Iterating through all the source files that appear in the Project Navigator of a project file, and ensures all are associated with a target within the same project/workspace. Any orphan or "dangling" files found will result in an error.  This is particularly useful for test files.

Reply:
s/Iterating/Iterates

Also, I'm not sure dangling needs quotes. If we feel it's necessary to explain what it is, perhaps we should just use "orphan" everywhere instead.

Reply:
I borrowed that terminology from docker. `docker images -qf dangling=true`